cli/command/service/update.go

```diff
@@ -459,6 +459,20 @@ func portConfigToString(portConfig *swarm.PortConfig) string {
 	return fmt.Sprintf("%v/%s", portConfig.PublishedPort, protocol)
 }
 
+func validatePort(val string) (string, error) {
```

Comment:
I would add to the name of this function some notion that is for port removal only.
Otherwise as it stands now, it gives the impression it can be reused to validate ports for any service cli.